edubots_surveys.Rnw

```diff
@@ -306,6 +306,12 @@ In the framework of the EduBots KA2 European project\footnote{https://www.edubot
 
 The technology behind chatbots \cite{Gong2008,bradesko2012} has become an usual element in our everyday life, having played an important role in the development of many fields, including education and online tutoring \cite{clarizia2018,Smutny2020}, which is what we are mainly concerned with. These automatic systems ease personalized learning, 
 adapting individuals' pace of learning, who get customized online tutoring outside the classroom. Thus, chatbots can significantly contribute to providing interactive learning experience as well as individual attention \cite{agarwal2020}.
+Given the potential benefits that chatbots could bring to the classroom, the focus of the EDUBOTS KA2 European project [1], the sponsor of this study, is to gather, analyse and disseminate best practice of the use of chatbots in higher education institutions.  This study involves two phases of data collection. The first phase aimed to collect feedback from students about their actual needs \cite{MoraChatbots2021}.  Hence, the objective was to collect information about students’ preferences of tools/applications for chatting and messaging, how they use them in educational context, who do they like to be with in the class messaging groups, and their expectations of the chatbots in terms of assisting them during their learning process. Therefore, two surveys for bachelor and master degree students at the University of Granada (Spain) were developed and answers from more than 250 students were collected.   The key findings of the students surveys reveal that they prefer to use messaging applications, such as Telegram or WhatsApp as they are familiar to them. Students also expressed interest in chatbots that can assist them with organizing their course schedules, help them access their assignment grades and facilitate searching for course’s resources. 
```

Comment:
Again, as below, I would really need you to delete whatever this substitutes. And please also edit the [1] and substitute if for whatever code it had while you're at it; don't worry if you don't, I can do it, but it would really help to have the original and changed text side by side.